pkgs/tools/filesystems/stratisd/default.nix

```diff
@@ -43,6 +44,22 @@ stdenv.mkDerivation rec {
     };
   };
 
+  patches = [
+    # Can be removed with the next release after v. 3.6.3
+    (fetchpatch {
+      name = "./remove-unused-imports.patch";
+      url = "https://github.com/stratis-storage/stratisd/commit/78440de6e6ed8eab5ddd25dbdfb7804d0698f2a2.patch";
+      hash = "sha256-RW2nyAWaoIbqrgbhCApQsMXkJWtWoOWL3VO7fIImJgY=";
+    })
+
+    # Can be removed with the next release after v. 3.6.3
+    (fetchpatch {
+      name = "./flag-import-not-used-in-build-as-test-only.patch";
```

Comment:
```suggestion
      name = "flag-import-not-used-in-build-as-test-only.patch";
```

Reply:
D'oh! Those were a remnant of me testing things with an offline patch first. Thanks!